x/auth/src/client/grpc.rs

```diff
@@ -46,7 +54,9 @@ where
         let req = AuthNodeQueryRequest::Account(request.into_inner().try_into()?);
         let response = self.app.typed_query(req)?;
         let response: AuthNodeQueryResponse = response.try_into()?;
-        let AuthNodeQueryResponse::Account(response) = response;
+        let AuthNodeQueryResponse::Account(response) = response else {
+            return Err(Status::unknown("invalid response enum variant"));
```

Comment:
We usually return an `internal` error here e.g. https://github.com/rumos-io/gears/blob/abe901d3886b32481eeeea7e7c9796bb9e2805e6/x/bank/src/client/grpc.rs#L46-L48

Reply:
done